packages/core/package.json

```diff
@@ -12,9 +12,9 @@
   "files": [
     "dist",
     "actor",
+    "actors",
```

Comment:
it seems to be super confusing that we have `xstate/actor` and `xstate/actors` here, the `/actor` one should be removed and it's utils should be moved to the root entry or something (it only contains `isActorRef` and `toActorRef`). Alternatively, those can be moved to `xstate/actors` but I probably prefer the first suggestion

Reply:
Yes, agreed. Will make this change